src/utils/util_conv.cc

```diff
@@ -138,6 +138,134 @@ drake::lcmt_iiwa_status ConvertToLcmStatus(
   return robot_status;
 }
 
+robot_msgs::robot_status_t ConvertToRobotStatusLcmMsg(
+    const franka_driver::RobotData& robot_data) {
+  const auto& robot_status {robot_data.robot_state};
+
+  robot_msgs::robot_status_t status_msg {};
+  int num_joints {static_cast<int>(robot_status.q.size())};
```

Comment:
use `auto` if you are already static casting. why cast to int at all and not just auto?